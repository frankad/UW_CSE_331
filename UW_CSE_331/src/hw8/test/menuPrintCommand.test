# Verify the print output option displayed by the
# menu command. It display the option on the menu when m is choosen.
m

# terminate the program
q