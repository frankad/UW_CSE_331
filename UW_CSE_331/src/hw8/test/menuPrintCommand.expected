Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify the print output option displayed by the
# menu command. It display the option on the menu when m is choosen.
Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): 
# terminate the program